src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$card-width: 200px;
+$card-height: 300px;
+$cards-in-row: 4;
+$cards-gap: 48px;
+$card-border-color: #ccc;
+$cards-container-width:
+  $card-width * $cards-in-row + $cards-gap * ($cards-in-row - 1);
```

Comment:
I can not find were you use it

Reply:
That was some experiment, forgot to delete it))